src/Compilers/CSharp/Portable/Symbols/TypeParameterSymbol.cs

```diff
@@ -690,10 +690,13 @@ internal override TypeSymbol MergeNullability(TypeSymbol other, VarianceKind var
 
         #region ITypeParameterTypeSymbol Members
 
-        CodeAnalysis.NullableAnnotation ITypeParameterSymbol.ReferenceTypeConstraintNullableAnnotation
+        CodeAnalysis.NullableAnnotation ITypeParameterSymbol.ReferenceTypeConstraintNullableAnnotation => ReferenceTypeConstraintIsNullable switch
         {
```

Comment:
>{ [](start = 8, length = 1)

Consider indenting the ```switch``` body and putting it on a separate line. Like so:
```
        CodeAnalysis.NullableAnnotation ITypeParameterSymbol.ReferenceTypeConstraintNullableAnnotation =>
            ReferenceTypeConstraintIsNullable switch
            {
                false when !HasReferenceTypeConstraint => CodeAnalysis.NullableAnnotation.NotApplicable,
                false => CodeAnalysis.NullableAnnotation.NotAnnotated,
                true => CodeAnalysis.NullableAnnotation.Annotated,
                null => CodeAnalysis.NullableAnnotation.Disabled,
            };
```

I think this way the code will be easier to follow.

Reply:
I didn't do this because currently the IDE doesn't have a good formatting story for switch expressions, and that won't be fixed until 16.1 brings in @sharwell's change in [https://github.com/dotnet/roslyn/pull/33938](https://github.com/dotnet/roslyn/pull/33938). I can format them like this and disable the formatting analyzer for this region if you'd prefer.

---
In reply to: [270182494](https://github.com/dotnet/roslyn/pull/34429#discussion_r270182494) [](ancestors = 270182494)